From b92b397d5e4ffe10ed92fb2fc6711758ef5bcf89 Mon Sep 17 00:00:00 2001
From: Tvrtko Ursulin <tvrtko.ursulin@intel.com>
Date: Tue, 03 Dec 2013 15:09:57 +0000
Subject: build: Make sure asserts are enabled for tests

Tests depend on assertions being enabled since they can, and do,
contain actual test steps. They are also mandatory for ensuring
sane test case behaviour.

Signed-off-by: Tvrtko Ursulin <tvrtko.ursulin@intel.com>
Signed-off-by: Daniel Vetter <daniel.vetter@ffwll.ch>
---
(limited to 'lib/check-ndebug.h')

diff --git a/lib/check-ndebug.h b/lib/check-ndebug.h
new file mode 100644
index 0000000..68a6ec4
--- /dev/null
+++ lib/check-ndebug.h
@@ -0,0 +1,3 @@
+#ifdef NDEBUG
+#error "Testsuite needs its asserts!"
+#endif
--
cgit v0.9.0.2-2-gbebe
